src/sdk/pynni/tests/assets/search_space.json

```diff
@@ -0,0 +1,74 @@
+{
+  "choice_str": {
+    "_type": "choice",
+    "_value": ["cat","dog", "elephant", "cow", "sheep", "panda"]
```

Comment:
I wonder why this folder name is "assets"?

Reply:
Because this is needed by the tests